src/arviz_plots/plots/ppctstat.py

```diff
@@ -0,0 +1,180 @@
+        raise TypeError(
+            "`group` argument must be either `posterior_predictive` or `prior_predictive`"
+        )
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    sample_dims = list(sample_dims)
+    if stats_kwargs is None:
+        stats_kwargs = {}
+    else:
+        stats_kwargs = stats_kwargs.copy()
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    else:
+        plot_kwargs = plot_kwargs.copy()
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    if aes_map is None:
+        aes_map = {}
+    else:
+        aes_map = aes_map.copy()
+
+    if data_pairs is None:
+        data_pairs = (var_names, var_names)
+    else:
+        data_pairs = (list(data_pairs.keys()), list(data_pairs.values()))
+
+    predictive_dist = process_group_variables_coords(
+        dt, group=group, var_names=data_pairs[0], filter_vars=filter_vars, coords=coords
+    )
+
+    predictive_dist = predictive_dist.stack(sample=sample_dims)
+    if t_stat == "median":
+        predictive_dist = predictive_dist.median(dim=list(predictive_dist.dims)[0])
+    elif t_stat == "mean":
+        predictive_dist = predictive_dist.mean(dim=list(predictive_dist.dims)[0])
+    elif t_stat == "std":
+        predictive_dist = predictive_dist.std(dim=list(predictive_dist.dims)[0])
+    else:
+        try:
+            t_stat_float = float(t_stat)
+        except ValueError as ve:
+            raise ValueError(f"T statistics '{t_stat}' not implemented") from ve
+        if 0 < t_stat_float < 1:
+            predictive_dist = predictive_dist.quantile(
+                q=t_stat_float, dim=list(predictive_dist.dims)[0]
+            )
+        else:
+            raise ValueError(f"T statistic '{t_stat}' not in valid range (0, 1).")
+    if plot_collection is None:
+        pc_kwargs["plot_grid_kws"] = pc_kwargs.get("plot_grid_kws", {}).copy()
+
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        pc_kwargs.setdefault("cols", "__variable__")
+        pc_kwargs.setdefault("rows", None)
+
+        pc_kwargs = set_figure_layout(pc_kwargs, plot_bknd, predictive_dist)
+
+        plot_collection = PlotCollection.wrap(
+            predictive_dist,
+            backend=backend,
+            **pc_kwargs,
+        )
+    # Plot KDE lines
+    plot_dist(
+        predictive_dist,
+        var_names=None,
+        group=None,
+        coords=None,
+        sample_dims=["sample"],
+        point_estimate=None,
+        ci_kind=None,
+        ci_prob=None,
```

Comment:
This should not be None. We should have these as top arguments and pass them here.

Related to this, not sure what to do with the following elements, should we default them to `False`? Or keep them. I am worried the plot could look overcrowded, although having the point estimate is informative. Maybe just default `point_estimate_text` to `False`?

* credible_interval
* point_estimate
* point_estimate_text